pkgs/by-name/ir/irrd/package.nix

```diff
@@ -119,8 +128,15 @@ py.pkgs.buildPythonPackage rec {
     asgi-logger
     wtforms-bootstrap5
     email-validator
+    jinja2
   ] ++ py.pkgs.uvicorn.optional-dependencies.standard;
 
+  postPatch = ''
```

Comment:
`postPatch` belong to patchPhase. So I'd move it together with `patches`, line 70.
`pythonRelaxDeps` is between patchPhase/configurePhase because it edits files, it's patching files. I'd add it after `postPatch`.
Hope it makes sense.